lib/blacklight/models/content.rb

```diff
@@ -9,7 +9,9 @@ def initialize(xml)
     end
 
     def canvas_conversion
-      "<a href='$IMS_CC_FILEBASE$/#{@linkname}'>#{@linkname}</a>"
+      # rubocop:disable LineLength
```

Comment:
Still, it is unreadable all on one line. Breaking into multiple lines makes it so I can see what that html says.